src/styles/theme-manager.js

```diff
@@ -191,8 +191,35 @@ module.exports = {
         backgroundColor: 'transparent',
         borderColor: rawTheme.palette.borderColor,
       },
+      isRtl: rawTheme.isRtl,
```

Comment:
You could just default `isRtl` to `false` here.

Then, if the user wishes to enable `isRtl`, you can add a function to `theme-manager` to override that and return a new `muiTheme` object with merged `styleConstants`.